pennylane/devices/default_qubit_torch.py

```diff
@@ -176,17 +176,23 @@ def circuit(x):
     _norm = staticmethod(torch.norm)
     _flatten = staticmethod(torch.flatten)
 
-    def __init__(self, wires, *, shots=None, analytic=None, torch_device="cpu"):
-        self._torch_device = torch_device
+    def __init__(self, wires, *, shots=None, analytic=None, torch_device=None):
+
+        if torch_device is None:
+            self._torch_device = "cpu"
+            self._user_def_torch_device = False
+        else:
+            self._torch_device = torch_device
+            self._user_def_torch_device = True
```

Comment:
Oh didn't know that, thank you! :slightly_smiling_face: One piece of information that's also being stored is `self._user_def_torch_device`. We'd like to know if the user has explicitly set a Torch device because the QNode creates a `default.qubit.torch` device under the hood. So maybe it's good to use the branching logic for that. :thinking: